pkg/storkctl/clusterpair.go

```diff
@@ -341,6 +474,24 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 	createClusterPairCommand.Flags().StringVarP(&destToken, "dest-token", "", "", "(Optional)Destination cluster token for cluster pairing")
 	createClusterPairCommand.Flags().StringVarP(&projectMappingsStr, "project-mappings", "", "", projectMappingHelpString)
 
+	// New parameters for creating backuplocation secret
+	createClusterPairCommand.Flags().StringVarP(&provider, "provider", "p", "", "Cloud provider type [s3,azure,google]")
+	createClusterPairCommand.Flags().StringVar(&bucket, "bucket", "", "Bucket name")
+	createClusterPairCommand.Flags().StringVar(&encryptionKey, "encryption-key", "", "Encryption key for backuplocation")
+	// AWS
+	createClusterPairCommand.Flags().StringVar(&s3AccessKey, "s3-access-key", "", "Accesskey for AWS")
+	createClusterPairCommand.Flags().StringVar(&s3SecretKey, "s3-secret-key", "", "Secret key for AWS")
```

Comment:
```suggestion
	createClusterPairCommand.Flags().StringVar(&s3SecretKey, "s3-secret-key", "", "AWS Secret key [AWS_SECRET_KEY]")
```

Reply:
As discussed modified it to "Secret Key for S3"